docs/sources/reference/commandline/cli.md

```diff
@@ -169,6 +169,7 @@ expect an integer, and they can only be specified once.
       -l, --log-level="info"                 Set the logging level
       --label=[]                             Set key=value labels to the daemon
       --log-driver="json-file"               Default driver for container logs
+	  --log-opt=[]                           Log driver specific options
```

Comment:
Sorry... looks like they still don't align (or is GitHub is rendering this incorrect?) :'(

Tabs/Spaces?